src/Exception/InvalidShapeException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+
+namespace Ipssi\Evaluation\Exception;
+
+
+class InvalidShapeException

```

Comment:
```suggestion
class InvalidShapeException extends \Exception
```
Sinon ce n'est pas une exception